zipkin-storage/cassandra3/src/main/java/zipkin/storage/cassandra3/CassandraUtil.java

```diff
@@ -96,14 +97,17 @@ public static int durationIndexBucket(long ts_micro) {
       return Collections.emptyList();
     }
     checkArgument(request.serviceName != null, "serviceName needed with annotation query");
-    Set<String> annotationKeys = new LinkedHashSet<>();
```

Comment:
went to list with manual dedupe to keep order. (i think the reason was to have some consistency when reading the data in `cqlsh`. ie you're always reading it as service, annotations, binaryAnnotations)